applications/python-the-coffee-bar-apps/src/utils/tracing.py

```diff
@@ -0,0 +1,70 @@
+from opentelemetry import trace
+from opentelemetry.exporter.jaeger import JaegerSpanExporter
+from opentelemetry.exporter.otlp.trace_exporter import OTLPSpanExporter
+from opentelemetry.exporter.zipkin import ZipkinSpanExporter
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import (
+    ConsoleSpanExporter,
+    SimpleExportSpanProcessor,
+    BatchExportSpanProcessor,
+)
+
+
+def configure_tracing(configuration: dict):
+    # OTLP Exporter configuration
+    if configuration['exporter'] == 'otlp':
+        service_name = {
```

Comment:
we could extract config building code to a separate methods so that if bodies would not be that fat.